### ISSUE TITLE
Темы эфира #154 (24 февраля)

### ISSUE THREAD

**Issue by akaguny** [znprod/stream-topics#76]

**Comment by akaguny** [comment]
Ребята в 13, 18 лет пишут свои фреймворки и библиотеки, а что делал ты?
https://github.com/talentlessguy (Россия) https://github.com/SiddharthShyniben (Индия) ...

**Comment by RinatValiullov** [comment]
> Ребята в 13, 18 лет пишут свои фреймворки и библиотеки, а что делал ты? [@talentlessguy](https://github.com/talentlessguy) (Россия) [@SiddharthShyniben](https://github.com/SiddharthShyniben) (Индия) ...

Типо того? "Я в твои годы уже старшим слесарем был, а чего добился ты, неудачник?" 😸

**Comment by RinatValiullov** [comment]
[Вселенная Стивена Вольфрама](https://habr.com/ru/post/649483)

**Comment by RinatValiullov** [comment]
[Марк и Влад критикуют криптоэкономику и NFT](https://youtu.be/9FUxp8WqzrQ?t=1560) (youtube)

**Comment by RinatValiullov** [comment]
[A Rust match made in hell](https://fasterthanli.me/articles/a-rust-match-made-in-hell)

**Comment by RinatValiullov** [comment]
Новость-вспышка. Подогреть фронтендерские пуканы.

[Tailwind CSS just broke 2 million downloads per week!](https://twitter.com/adamwathan/status/1483187325043720196?s=20&t=6-zyBontv35iTdf7qqWd3Q)

**Comment by RinatValiullov** [comment]
[Blazor: Нужен ли нам .Net в вебе?](https://habr.com/ru/company/orioninc/blog/651311)

**Comment by RinatValiullov** [comment]
[Рейтинг языков программирования 2022. C# обошел Java, TypeScript сравнялся с PHP, а Dart – наиболее комфортный язык](https://habr.com/ru/post/651585)

**Comment by akaguny** [comment]
@RinatValiullov да, plantuml, которого мы заслуживаем. Интересно использовать со всякими там marp, revealjs, remark..  для презентаций в markdown

**Comment by ghost** [comment]
Google представил Chrome OS Flex, пригодный для установки на любое оборудование 

https://www.opennet.ru/opennews/art.shtml?num=56706

**Comment by RinatValiullov** [comment]
@akaguny Не только 😉 

[![image](https://user-images.githubusercontent.com/9755333/154350820-640a8292-82c7-4a7e-aa07-a1e0e54ed17f.png)](https://mermaid.live/view/#pako:eNpNkLFOAzEMhl8l8ty8wA0wcGVGgommg2lcGuCSKuSEUNUBGBhgYEaCZzgJIRXK8QzOG9XlinSZnN-fP8lewCRYggLOI85n6qQ0Xsk7GPFrvudfle-44Z_8xF9SSrLidX7mtgse8iM3f3ErjW916o9isPvjnUNpvVeOplhMUd84T_ospHRFil9EIHJu--RwR9bXFLV3_gIVv221vBK46aOHfTQ5Mb7zmj_5YwwDqChW6KystNiOGEgzqshAIaXFeGnA-KVw9dxioqF1KUQoUqxpAFincHzrJ___jikdynWqLlxuANeIgSs)

**Comment by RinatValiullov** [comment]
[State of JS](https://2021.stateofjs.com/ru-RU)
